### ISSUE TITLE
Displaying only Work Items parents when using "Generate Release Notes (Crossplatform)" extension.

### ISSUE THREAD

**Issue by TechWatching** [rfennell/AzurePipelines#776]
**Your question? Please describe.**
Is it possible to only display parents of work items on the report ?
In Azure DevOps we are associating Tasks to commits / PR but tasks only comport technical information which is not relevant for our release note. However, parents of the tasks which are the Product Backlof Items are the items we want to display in our release note. Is it possible to display them without the associated tasks and only once.

**Example**
* Tasks associated to commits : 
  * Task A1 (parent : PBI A)
  * Task A2 (parent : PBI A)
  * Task B1 (parent : PBI B)
  * Task B2 (parent : PBI B)
  * Task B3 (parent : PBI B)
* Expected release notes :
```
Product Backlog Items
- PBI A
- PBI B
```

**Releated issues**
I have seen other related issues like the #690 but I don't think it allows to obtain the expected release notes above.

**Details**
* Azure DevOps Services
* Generate Release Notes (Crossplatform) v3.3.10

**Comment by rfennell** [comment]
Sorry for not getting back sooner, I have been on annual leave. The answer to you question is yes, it is possible to list just parents. You would need to loop across the associated work items, but not write them but, then have an inner loop finding parents.

There is another issue open about parent WI #777 - I will work out the template for your needs whilst looking into that one

**Comment by TechWatching** [comment]
No problem. Thanks for your answer. Do you think it is possible without repeating the parents multiple times (as in my sample where associated WI have same parent) ?

**Comment by rfennell** [comment]
I think it will be possible to do it without repeats, but will probably need a custom Handlebars extension (a dictionary is my I guess) to remove duplicates

**Comment by rfennell** [comment]
I have added a sample Handlebars Helper to list just the parents of associated workitems

```javascript
/* Usage
# Just Parents
{{#return_parents_only this.workItems this.relatedWorkItems}}
  - {{this.id}} - {{lookup this.fields 'System.Title'}} 
{{/return_parents_only}} 
*/

module.exports = {
    return_parents_only(array, relatedWorkItems, block) {
        var ret = ''; 
        var foundList = [];
        for (var arrayCount = 0; arrayCount < array.length ; arrayCount++) {
             for (var relationCount = 0; relationCount < array[arrayCount].relations.length; relationCount++) { 
                if (array[arrayCount].relations[relationCount].attributes.name == 'Parent') { 
                    var urlParts = array[arrayCount].relations[relationCount].url.split("/");
                    var id = parseInt(urlParts[urlParts.length - 1]);
                    var parent = relatedWorkItems.find(element => element.id === id);
                    if (!foundList.includes(parent)) {
                        foundList.push(parent);
                        ret += block.fn(parent);
                    }
                }
            }
        }; 
        return ret;
    }
}
```

So pass in this code as a string in the `customHandlebarsExtensionCode` parameter
